app/javascript/Snackbar/SnackbarItem.jsx

```diff
@@ -1,7 +1,7 @@
 import { h } from 'preact';
 import PropTypes from 'prop-types';
-import { Button } from '@crayons';
 import { defaultChildrenPropTypes } from '../common-prop-types';
+import { Button } from '@crayons';
```

Comment:
This is an automatically fixed lint rule, so as mentioned in #8880, this causes no burden on a contributor.